src/Functions/array/arrayFill.cpp

```diff
@@ -128,8 +128,59 @@ using FunctionArrayReverseFill = FunctionArrayMapped<ArrayFillImpl<true>, NameAr
 
 REGISTER_FUNCTION(ArrayFill)
 {
-    factory.registerFunction<FunctionArrayFill>();
-    factory.registerFunction<FunctionArrayReverseFill>();
+    FunctionDocumentation::Description description = R"(
+The `arrayFill` function sequentially processes a source array from the first element
+to the last, evaluating a lambda condition at each position using elements from
```

Comment:
I think we should write just "using elements from the source array".
(I don't quite get what the "condition array" is within this function)

EDIT: Ignore me. I am wrong, I should have read on.